Formula/apt-dater.rb

```diff
@@ -14,12 +14,15 @@ class AptDater < Formula
     sha256 "2ac3ba56f32d018a9af477484d8ad561871f855aca78726dbe8f43f5552f6acc" => :yosemite
   end
 
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
   depends_on "pkg-config" => :build
   depends_on "gettext"
   depends_on "glib"
   depends_on "popt"
 
   def install
+    system "autoreconf", "-ivf"
```

Comment:
Does the prebuilt configure script not work in this release?

Reply:
No, it errors out about missing `install-sh` and things like that. There is a `install-sh` provided in the download, but it turns out just to be a broken symlink. Running `autoreconf` seemed to set everything straight.